tap_linkedin_ads/streams.py

```diff
@@ -617,7 +617,8 @@ class VideoAds(LinkedInAds):
     parent = "accounts"
     params = {
         "q": "dscAdAccount",
-        "dscAdTypes": "List(VIDEO)"
+        "dscAdTypes": "List(VIDEO)",
+        "count":100
```

Comment:
What's the max limit and the default value?

Reply:
max limit is 100 and default is 10, 
Page size configured can override this, hence we need to hardcode the value.